sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -0,0 +1,792 @@
+## API Report File for "@azure/ai-text-analytics"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AzureKeyCredential } from '@azure/core-auth';
+import { CommonClientOptions } from '@azure/core-client';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationOptions } from '@azure/core-client';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export interface ActionCommon {
+    disableServiceLogs?: boolean;
+}
+
+// @public
+export type ActionPrebuilt = ActionCommon & {
+    modelVersion?: string;
+};
+
+// @public
+export const AnalyzeActionName: readonly ["EntityLinking", "EntityRecognition", "KeyPhraseExtraction", "PiiEntityRecognition", "LanguageDetection", "SentimentAnalysis"];
+
+// @public
+export type AnalyzeActionName = typeof AnalyzeActionName[number];
```

Comment:
I'm a little confused why we need to get the typeof the array and we can't use a union of string literals?

Reply:
We could but I mainly want to have that information in one place (the array) so when new things are added, they can just be added to that array.

Reply:
I replaced the array with an object and used it in the tests.